q2_dada2/_denoise.py

```diff
@@ -99,6 +99,12 @@ def _denoise_helper(biom_fp, track_fp, hashed_feature_ids):
     df = pd.read_csv(track_fp, sep='\t', index_col=0)
     df.index.name = 'sample-id'
     df = df.rename(index=_filepath_to_sample)
+
+    if 'merged' in df:
+        percentage_merged = [round(m / d * 100, 2) for m,
```

Comment:
I think that the more idiomatic pandas approach would involve not iterating over the rows, but rather just performing column manipulations. For example:

```python
x = pd.DataFrame([[1,2,3],[4,5,6]], columns=["foo", "bar", "baz"], index=["s1", "s2"])
x["xyz"] = x["foo"] / x["bar"] # adds new column, "xyz"
x = x[["foo", "bar", "xyz", "baz"]] # reorders columns
```